themes/hummingbird/hooks-hummingbird-theme.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Hooks on Hummingbird Theme
+weight: 3
+---
+
+# Hooks on Hummingbird Theme
+
+Hummingbird theme introduced a brand new UI for the PrestaShop front office. 
+
+A visual map of hook locations was created by the Design team at PrestaShop. 
+
+{{% notice note %}}
+This page integrates Figma iframes, it may require heavy resources and cause performance issues on slower machines.
+{{% /notice %}}
+
+{{< toc >}}
+
+## Home page
```

Comment:
Let's add a buttons "View in the separate tab" above every view, the other option would be to list all the pages as a separate .md files. What do you think? The number of pages is finite so it would make sense.

Reply:
i splitted by section in separates md
looks better and lighter also